src/calc/phased_payment_calculator.py

```diff
@@ -42,10 +42,10 @@ def __init__(self, founders_map, owners_map, service_fee_calculator, min_delegat
     # owners reward = owners payment = total reward - delegators reward
     # founders reward = delegators fee = total reward - delegators reward
     ####
-    def calculate(self, reward_provider_model):
+    def calculate(self, reward_provider_model, total_rwrd_amnt):
```

Comment:
Do we need to pass it as an argument? We could just do:
```python
total_rwrd_amnt = reward_provider_model.total_reward_amount
```
locally inside the function.